Move the remaining RN wiki content to website (#3120)
facebook/react-native-website: website/contributing/contributing-intro.md

We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).

### Repositories

The main repository contains the React Native framework itself, and it is here where we keep track of bug reports and manage pull requests.

There are a few other repositories you might want to familiarize yourself with:

- **React Native website** which contains the source code for the website, including the documentation, located [in this repository](https://github.com/facebook/react-native-website).
- **Releases** Conversations for new releases are happening [in this discussion repo](https://github.com/reactwg/react-native-releases/discussions).
- **Changelog** The changelog can be found [here](https://github.com/facebook/react-native/blob/main/CHANGELOG.md).
- **Discussions** about the future of React Native take place in the <https://github.com/react-native-community/discussions-and-proposals> repository.

> - **High-quality plugins** for React Native can be found throughout the [React Native Community GitHub Organization](http://github.com/react-native-community/).

Reviewer: @Simek the community plugins have been moved out into separate organizations and projects, so this line is no longer relevant

Author: ```suggestion
- **High-quality plugins** for React Native can be found throughout the [React Native Directory](https://reactnative.directory) website.
```

@cortinico What do you think about adding mention for the RN Directory here instead?

---

> title: Contributing Intro

Author: It was the earlier name of the page, but I feel like it doesn't suit the content that good, also it looks odd with other How To's in the sidebar. 

The idea behind the name was to reflect the summary/overview content in there. What do you think about, for example: "Contributing Overview"? But I'm not sure if this is more formal though, however feel free to post you ideas, I'm not that good at naming. 😅

> title: Contributing Intro

Author: Maybe we can go with "Overview", since we already have one for the Architecture: 
* https://reactnative.dev/architecture/overview

However, I also like the "Contributing Introduction" suggestion, it sounds a bit more formal to me. 

Let's wait what @cortinico prefer, then I update the title, filename, links and redirect.

> title: Contributing Intro

Reviewer 2: I'm fine either way. Just "Contributing" as a landing page title also works as it's a known keyword in the OSS space (i.e. you have a CONTRIBUTING.md file).

Otherwise "Contributing Overview" works fine for me 👍

3. **Help people write test plans.** 

   Some pull requests sent to the main repository may lack a proper test plan. These help reviewers understand how the change was tested, and can speed up the time it takes for a contribution to be accepted.

Each of these tasks is highly impactful, and maintainers will greatly appreciate your help.

### Our Development Process

We use GitHub issues and pull requests to keep track of bug reports and contributions from the community. All changes from engineers at Meta will sync to [GitHub](https://github.com/facebook/react-native) through a bridge with Meta's internal source control. Changes from the community are handled through GitHub pull requests. 

Once a change made on GitHub is approved, it will first be imported into Facebook's internal source control and tested against Facebook's codebase. Once merged at Facebook, the change will eventually sync back to GitHub as a single commit once it has passed Facebook's internal tests.

You can learn more about the contribution process in the following documents:

- [Triaging GitHub Issues](/contributing/triaging-github-issues)
- [Managing Pull Requests](/contributing/managing-pull-requests)

> We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to us through [@ReactNative](http://twitter.com/reactnative) (the React Native team) and [@ReactNativeComm](http://twitter.com/reactnativecomm) (the React Native Community organization).

Reviewer 3: ```suggestion
We also have a thriving community of contributors who would be happy to help you get set up. You can reach out to the React Native team through [@ReactNative](http://twitter.com/reactnative).
```

The react native community twitter account has been very inactive for quite some time

Author: Oh, let's remove that mention for now.

4. If you've changed APIs, update the documentation, which lives in [separate repo](https://github.com/facebook/react-native-website/).
5. Ensure the test suite passes, either locally or on CI once you opened a pull request.
6. Make sure your code lints (for example via `yarn lint --fix`).
7. Push the changes to your fork.
8. Create a pull request to the React Native repository.
9. Review and address comments on your pull request.
10. A bot may comment with suggestions. Generally we ask you to resolve these first before a maintainer will review your code.
11. If changes are requested and addressed, please [request review](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify reviewers to take another look.
12. If you haven't already, please complete the [Contributor License Agreement](/contributing/contribution-license-agreement) ("CLA"). **[Complete your CLA here.](https://code.facebook.com/cla)**

If all goes well, your pull request will be merged. If it is not merged, maintainers will do their best to explain the reason why.

### Step-by-step Guide

> Whenever you are ready to contribute code, check out our [step-by-step guide to sending your first pull request](/contributing/how-to-open-a-pull-request), or read the [How to Contribute Code](/contributing/how-to-contribute-code) wiki for more details.

Reviewer 2: ```suggestion
Whenever you are ready to contribute code, check out our [step-by-step guide to sending your first pull request](/contributing/how-to-open-a-pull-request), or read the [How to Contribute Code](/contributing/how-to-contribute-code) page for more details.
```
